desktop/src/main/java/bisq/desktop/main/funds/withdrawal/WithdrawalView.java

```diff
@@ -629,6 +631,17 @@ public void updateItem(final WithdrawalListItem item, boolean empty) {
                     }
                 });
     }
+
+    private Coin getDust(Transaction tx) {
+        Coin dust = Coin.ZERO;
+        for (TransactionOutput txo: tx.getOutputs()) {
+            if (txo.getValue().value < 546) {
+                dust = dust.add(txo.getValue());
+                log.info("dust TXO = {}", txo.getValue().toFriendlyString());
```

Comment:
As an avid log reader, a bit more context will really help understand what's going on here.

Reply:
Done:

    Mar-28 21:25:47.953 [JavaFX Application Thread] INFO  b.d.m.f.w.WithdrawalView: dust TXO = TxOut of 0.000001 BTC to msskpXKKdJdQL82zB76VRU5o5qcoXu1aYf script:DUP HASH160 PUSHDATA(20)[8790aa8150dd74fe091af8253760a23772d2e62a] EQUALVERIFY CHECKSIG 
    Mar-28 21:25:47.953 [JavaFX Application Thread] INFO  b.d.m.f.w.WithdrawalView: Dust output (100 satoshi) was detected, the dust amount has been added to the fee (was 4068, now 4168)